demo/lib/host.bash

```diff
@@ -197,3 +197,12 @@ host-delete-vm() {
         command-error "deleting govm \"$VM_NAME\" failed"
     }
 }
+
+host-is-encrypted-ssh-key() {
+    SSH_ASKPASS=/bin/false ssh-keygen -y -f "$1" < /dev/null 2>/dev/null
```

Comment:
> * replacing `2>/dev/null` by `&>/dev/null`/ would clean up ssh key being printed from test startup.

Yes, that makes sense in `host-is-encrypted-key()`. I'll update.

> * regardless of `SSH_ASKPASS` and `< /dev/null`, `ssh-keygen` keeps asking password when `DISPLAY` is not set.
> * Possible solutions to make sure `ssh-keygen` and `ssh-add` will never ask passwords:
>   
>   * `export SSH_ASKPASS=/bin/false SSH_ASKPASS_REQUIRE=force` -- maybe we could have these in the beginning of `command.bash`?

This only works with pretty recent versions of `ssh` which do recognize the `SSH_ASKPASS_REQUIRE` environment variable. And then with such versions it would make more sense to use `SSH_ASKPASS_REQUIRE=never`.

But all things considered, I think the best is to just force DISPLAY to a bogus value instead. So something like
```
DISPLAY=none SSH_ASKPASS=/bin/false ssh-keygen -y -f $SSH_KEY < /dev/null
```
should do the trick for all current versions.